src/components/views/Experiment.js

```diff
@@ -102,8 +102,25 @@ export default class extends React.Component {
             <div>
                 <article id="experiment">
                     <h2>{this.props.name || this.props.slug}</h2>
+                    <section id="experiment-details" className={maybeDescription ? 'with-description' : 'without-description'}>
+                        <h3>Details</h3>
+                        {maybeDescription}
+                        <section id="experiment-counts">
+                            <h4>Counts</h4>
+                            <section id="experiment-client-counts">
+                                <h5>Clients</h5>
+                                {this.getCountDL(this.props.populations, 'total_clients')}
+                            </section>
+                            <section id="experiment-ping-counts">
+                                <h5>Pings</h5>
+                                {this.getCountDL(this.props.populations, 'total_pings')}
+                            </section>
+                        </section>
+                        {maybeAuthors}
+                    </section>
                     <aside id="experiment-options">
                         <h3>Options</h3>
+                        <input type="text" placeholder="Search..." onKeyUp={this.props.onSearch} />
```

Comment:
`type="search"` and it will default to text for older browsers....which will fail on CSS grid anyway 😁

Reply:
Oh man. I can't believe I didn't know about this. Updated!